### PR TITLE
Use relative path with working directory (fixes # 1515)

### PR DESCRIPTION
**Pull request by azhidkov**
Fix for this error https://github.com/go-lang-plugin-org/go-lang-idea-plugin/issues/1515

### CODE REVIEW

**Comment by azhidkov** [comment]
Image with configuration window http://postimg.org/image/nl0sjv0ez/

**Comment by zolotov** [comment]
Looks good but
1. what about completion in `file` text field (see com.goide.runconfig.GoRunUtil#installGoWithMainFileChooser)
2. it makes sense to add the same feature in test-file and test-directory run configurations (GoTestRunConfiguration)

File: src/com/goide/runconfig/GoRunConfigurationWithMain.java [hunk 1]
```diff
@@ -60,7 +60,11 @@ public void writeExternal(Element element) throws WriteExternalException {
   protected void checkFileConfiguration() throws RuntimeConfigurationError {
     VirtualFile file = VirtualFileManager.getInstance().findFileByUrl(VfsUtilCore.pathToUrl(getFilePath()));
     if (file == null) {
-      throw new RuntimeConfigurationError("Main file is not specified");
+      // check relative path based on working directory
```
Comment (zolotov) [inline src/com/goide/runconfig/GoRunConfigurationWithMain.java]: redundant comment

File: src/com/goide/runconfig/GoRunConfigurationWithMain.java [hunk 2]
```diff
@@ -60,7 +60,11 @@ public void writeExternal(Element element) throws WriteExternalException {
...
+      String relativePath = FileUtil.join(getWorkingDirectory(), getFilePath());
```
Comment (zolotov) [inline src/com/goide/runconfig/GoRunConfigurationWithMain.java]: It's not relativePath, it's just path. Please rename.

**Comment by azhidkov** [comment]
I added the same feature for file (http://postimg.org/image/6olizuvf1/) and directory (http://postimg.org/image/om7mvsitf/) and fixed your comments about first commit. Plus ocassionally added fix for #1758 issue. 

But didn't realize what to do with (how it should be?):
> 1. what about completion in file text field (see com.goide.runconfig.GoRunUtil#installGoWithMainFileChooser)

**Comment by dlsniper** [comment]
I'd say please keep the fixes separated as it's easier to review them / get them merged. Thank you for your contribution ;)

**Comment by zolotov** [comment]
> Plus ocassionally added fix for #1758 issue.

That's funny, thanks a lot :) As @dlsniper said it's simpler to merge two separate PR for these two features.

> But didn't realize what to do with (how it should be?):

Don't know yet how it should be. Let's skip this, I'll try to think something of and maybe create a separate issue.

**Comment by azhidkov** [comment]
I forgot create separate branch for inspection, sorry :)
And fixed problem with two tests.

**Pull request merged**